### ISSUE TITLE
Show translation competition percentages

### ISSUE THREAD

**Issue by Pertempto** [arrocke/gloss-translation#211]
We didn't implement this in #171, but we don't want to forget it. 

The original requirement was
> The user also needs to be informed how complete the translation is.

This is blocked by #208

**Comment by Pertempto** [comment]
Should we show percentages? I think it would be more useful to see some kind of book-by-book breakdown of which translation will be used for each book, given the ordering of translations setup for the language.

Should we show this on the manage language page? Or should it be on a separate page, with some kind of link on the manage language page, maybe "View translation coverage"?

**Comment by arrocke** [comment]
I'm wanting to mock something up in figma for this. We want this progress data to be public so that everyone can see our progress.

**Comment by arrocke** [comment]
This was broken out into two separate issues:
* #357 
* #356

**Comment by arrocke** [comment]
Closing this in favor of #411
